go.mod

```diff
@@ -0,0 +1,12 @@
+module gitlab.cee.redhat.com/rhprod/rhprod-cli
+
+go 1.13
+
+require (
+        github.com/mitchellh/go-homedir v1.1.0
+        github.com/spf13/afero v1.1.2
+        github.com/spf13/cobra v0.0.5
+        github.com/spf13/viper v1.6.2
+        gitlab.sat.engineering.redhat.com/rhprod/rhprod-cli v1.4.0 // indirect
```

Comment:
I vote to scrub references to internal gitlab stuff (and rhprod).

Can you accomplish your goal without the rhprod references?